src/backend/gog/games.ts

```diff
@@ -112,11 +112,12 @@ class GOGGame extends Game {
       installPlatform
     )
     if (!info) {
-      logError(
+      logWarning(
         [
           'Could not get install info for',
-          `${this.appName},`,
-          'returning empty object. Something is probably gonna go wrong soon'
+          `${this.appName}`,
+          `using ${installPlatform} as platform,`,
+          'trying again with as Windows.'
```

Comment:
This can be confusing, if getInstallInfo fails when Windows is the selected platform this will say `Could not get install info for ... using Windows as platform, trying again with as Windows.`, and the retry is not explicit in the code (you don't know when/where is that retry happening unless you know the discord conversations).

Just saying it failed for X platform should be enough it it needs debugging, there will be another get info command with Windows as platform anyway in the log

Reply:
Good point, wil change tomorrow.

Reply:
done